public/app/plugins/datasource/loki/datasource.test.ts

```diff
@@ -639,22 +674,6 @@ describe('LokiDatasource', () => {
         it('then the correct label should be added for metrics query', () => {
           assertAdHocFilters('rate({bar="baz"}[5m])', 'rate({bar="baz", job="grafana"}[5m])', ds);
         });
-
```

Comment:
Is there a reason why are these removed?

Reply:
No, good catch, it was a bad merge.

Reply:
Added back.